crates/bevy_render/src/color/mod.rs

```diff
@@ -1083,69 +1083,184 @@ impl Color {
             }
         }
     }
-}
 
-impl Default for Color {
-    fn default() -> Self {
-        Color::WHITE
+    /// New `Color` from `[f32; 4]` in sRGB colorspace.
+    #[inline]
+    pub fn rgba_from(arr: impl Into<[f32; 4]>) -> Self {
```

Comment:
I think that the `rgb_from` is a bit awkward. IMO stick with `rgba_from_array` etc for all these method names :)